docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -0,0 +1,129 @@
+include::./attributes.adoc[]
+= {project-name} - Tips for writing native applications
+
+This guide contains various tips and tricks for getting around various problems that might arise when attempting to run java apps as native images.
```

Comment:
`around various problems` let's remove the `various` here, you already have one at the beginning of the sentence and it doesn't bring much.